sonar-scm-git-plugin/pom.xml

```diff
@@ -18,9 +18,8 @@
   </scm>
  
   <properties>
-    <!-- Minimal supported version of SonarQube. It is defined in travis.sh 
-      for integration testing too. -->
-    <sonar.apiVersion>7.7-alpha1</sonar.apiVersion>
+    <!-- Minimal supported version of SonarQube -->
+    <sonar.apiVersion>7.9</sonar.apiVersion>
```

Comment:
Shouldn't you change `sonarQubeMinVersion` to 7.9 then?

Reply:
good catch